coins/ltz.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "litecoinz",
+    "symbol": "ltz",
+    "algorithm": "equihash",
```

Comment:
The `sapling` key allows a block height as the value. Can you add it here for the activation height on mainnet?